### ISSUE TITLE
fromFormat and fromFromatExplain does not support Quarter values

### ISSUE THREAD

**Issue by el** [moment/luxon#595]
```
DateTime.fromFormatExplain("2017Q1","yyyyQq")
```
and
```
DateTime.fromFormatExplain("2017Q01","yyyyQqq")
```
should create a date without a problem. But they don't.

**Comment by icambron** [comment]
Quarters aren't currently supported in the parser, which isn't completely symmetrical with the formatter. You can see the table of tokens here: https://moment.github.io/luxon/docs/manual/parsing.html#table-of-tokens

I think this is a reasonable feature request, though. I'd take a pull request adding quarter support.

**Comment by ibarrae** [comment]
Hey folks @el @icambron , given the examples above, what should be the expected outcome?

Maybe it's something like this?
`DateTime.fromFormatExplain("2017Q1", "yyyyQq") === 2017-01-01T00:00:00.00:TZ`

**Comment by icambron** [comment]
You want to change `fromFormat()`; `fromFormatExplain` is a debugging tool. Also, it will parse to a local time by default. So you want something like:

```
DateTime.fromFormat("2017Q1", "yyyy'Q'q", { zone: "utc" }).toISO() //=> "2017-01-01T00:00.00Z"
```

**Comment by el** [comment]
I think `luxon` should have a configuration for quarters. It is common practice to have a different definition for quarters. What you are suggesting is called [Calendar Year](https://en.wikipedia.org/wiki/Calendar_year), [Fiscal Year](https://en.wikipedia.org/wiki/Fiscal_year) has a different start/end dates.

**Comment by icambron** [comment]
Luxon will not support customizable fiscal calendars, sorry.

**Comment by icambron** [comment]
Fixed by #602
